src/Badge.js

```diff
@@ -0,0 +1,73 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { space } from 'styled-system'
+import { theme } from '../'
+
+function type (props) {
+  const badgeColors = {
+    'green': {
+      backgroundColor: theme.colors.green,
+      color: theme.colors.white
+    },
+    'green-light': {
```

Comment:
Would be good to follow the same naming convention used in #79 – e.g. [`lightGreen`](https://github.com/pricelinelabs/design-system/pull/79/files#diff-e179cfde519bf6bae73373ad10d6ae2fR45)